qml/FirmwareUpdate.qml

```diff
@@ -23,7 +23,8 @@ Item {
             running = true
             flashProgress.value = progress
         }
-        onMessageChanged: {
+        onStateChanged: {
+            print("Flash state:", state)
```

Comment:
stale debug?

Can you give a longer commit message, it's vague right now.

Reply:
Removed.